cmd/depository/main.go

```diff
@@ -52,7 +52,7 @@ var (
 
 	// flags for depository certificate generation
 	templateImagePath = flag.String("cert-template-image", "resource/certificate_template.jpg", "template image for depository's certificate generation")
-	ttfFontPath       = flag.String("cert-ttf-font", "resource/ttf/stsong.ttf", "ttf font file for depository's certificate generation")
+	ttfFontPath       = flag.String("cert-ttf-font", "resource/ttf/SourceHanSansCN-Normal.ttf", "ttf font file for depository's certificate generation")
```

Comment:
The default path is `resource/ttf/SourceHanSansCN-Normal.ttf`, But in the image, our executable file is under `/bin` and the template files are under `/bc-saas/resource`, so the code should panic if it is run without the relevant parameters.

Reply:
The executable files in the image should also be placed under `/bc-saas`.

Reply:
that's fine. Its deployment relevant.